### DISCUSSION TITLE
Add support for displaying images/stickers in the terminal using ueberzugpp

### DISCUSSION THREAD

**Discussion by GitWloda** [d99kris/nchat#149] [Ideas]
I think it could be interesting to add support for viewing photos, video tumbnails and stickers with ueberzug

**Comment by d99kris** [comment]
Thanks for suggesting. Adding support for viewing photos/etc in the terminal has been suggested before as well - see #99. Unfortunately it's currently not in plans to be added, but will update here if it changes.

**Comment by d99kris** [comment]
Let's track the feature request under #99 instead. I will proceed to close this one.
